duckari/duck.py

```diff
@@ -25,13 +26,25 @@ def decorate(func: _ListenerT) -> _ListenerT:
 
         return decorate
 
+    def command(self, name: str = None): # TODO: Add return type
+        def decorate(func: _ListenerT) -> _ListenerT:
+            self._commands[name].append(func) # You can add multiple callbacks for a command.
```

Comment:
Why can a command have multiple callbacks??

Reply:
Because I felt like adding that, could be useful.

Reply:
idk what are the pros and cons?

Reply:
I'm fine with removing it if everyone else it (which it seems like)